src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -36,28 +36,28 @@ public class AddCommand extends Command {
             + PREFIX_TAG + "friends "
```

Comment:
Can consider to have a "female"/"male" tag. As the avatar will change based on the gender.